README.md

```diff
@@ -111,14 +111,22 @@ SQLx is compatible with the [`async-std`] and [`tokio`] runtimes.
 # Cargo.toml
 [dependencies]
 sqlx = "0.3"
+futures = "0.3"
+async-std = { version = "1.5.0", features = [ "attributes" ] }
```

Comment:
I'd rather leave it to explain their preferred method of installation - https://github.com/async-rs/async-std/#installation